tools/clang/test/CodeGenHLSL/batch/declarations/resources/textures/feedback.hlsl

```diff
@@ -24,43 +24,43 @@ float main() : SV_Target
     // CHECK: call void @dx.op.writeSamplerFeedbackLevel(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float 6.000000e+00)
-    feedbackMinLOD.WriteSamplerFeedbackLevel(texture2D, samp, coords2D, lod);
+    feedbackMinMip.WriteSamplerFeedbackLevel(texture2D, samp, coords2D, lod);
     // CHECK: call void @dx.op.writeSamplerFeedbackGrad(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float 7.000000e+00, float 8.000000e+00, float 4.000000e+00)
-    feedbackMinLOD.WriteSamplerFeedbackGrad(texture2D, samp, coords2D, ddx, ddy, clamp);
+    feedbackMinMip.WriteSamplerFeedbackGrad(texture2D, samp, coords2D, ddx, ddy, clamp);
     
     // Test with undef clamp
     // CHECK: call void @dx.op.writeSamplerFeedback(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float undef)
-    feedbackMinLOD.WriteSamplerFeedback(texture2D, samp, coords2D);
+    feedbackMinMip.WriteSamplerFeedback(texture2D, samp, coords2D);
     // CHECK: call void @dx.op.writeSamplerFeedbackBias(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float 5.000000e+00, float undef)
-    feedbackMinLOD.WriteSamplerFeedbackBias(texture2D, samp, coords2D, bias);
+    feedbackMinMip.WriteSamplerFeedbackBias(texture2D, samp, coords2D, bias);
     // CHECK: call void @dx.op.writeSamplerFeedbackGrad(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float 7.000000e+00, float 8.000000e+00, float undef)
-    feedbackMinLOD.WriteSamplerFeedbackGrad(texture2D, samp, coords2D, ddx, ddy);
+    feedbackMinMip.WriteSamplerFeedbackGrad(texture2D, samp, coords2D, ddx, ddy);
 
     // Test on every FeedbackTexture variant
     // CHECK: call void @dx.op.writeSamplerFeedback(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float undef)
-    feedbackTiled.WriteSamplerFeedback(texture2D, samp, coords2D);
+    feedbackMipRegionUsed.WriteSamplerFeedback(texture2D, samp, coords2D);
     // CHECK: call void @dx.op.writeSamplerFeedback(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float 3.000000e+00, float undef)
-    feedbackMinLODArray.WriteSamplerFeedback(texture2DArray, samp, coords2DArray);
+    feedbackMinMipArray.WriteSamplerFeedback(texture2DArray, samp, coords2DArray);
     // CHECK: call void @dx.op.writeSamplerFeedback(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float 3.000000e+00, float undef)
-    feebackTiledArray.WriteSamplerFeedback(texture2DArray, samp, coords2DArray);
+    feebackMipRegionUsedArray.WriteSamplerFeedback(texture2DArray, samp, coords2DArray);
 
     // Test with overloaded texture type
     // CHECK: call void @dx.op.writeSamplerFeedback(
     // CHECK: float 1.000000e+00, float 2.000000e+00, float undef, float undef)
-    feedbackMinLOD.WriteSamplerFeedback(texture2D_float4, samp, coords2D);
+    feedbackMinMip.WriteSamplerFeedback(texture2D_float4, samp, coords2D);
```

Comment:
We have to add validation to prevent this (when we can detect), but this case is not legal, since it's undefined to use the same feedback resource with two different texture resources.